plugins/dynatrace/README.md

```diff
@@ -51,6 +53,23 @@ metadata:
 
 The `DYNATRACE_ENTITY_ID` can be found in Dynatrace by browsing to the entity (a service, synthetic, frontend, workload, etc.). It will be located in the browser address bar in the `id` parameter and has the format `ENTITY_TYPE-ENTITY_ID`, where `ENTITY_TYPE` will be one of `SERVICE`, `SYNTHETIC_TEST`, or other, and `ENTITY_ID` will be a string of characters containing uppercase letters and numbers.
 
+##### Viewing Recent Synthetics Results
+
+To show recent results from a Synthetic Monitor, add the following annotation to `catalog-info.yaml`:
+
+```yaml
+# catalog-info.yaml
+# [...]
+metadata:
+  annotations:
+    dynatrace.com/dynatrace-synthetics-ids: SYNTHETIC_ID, SYNTHETIC_ID_2, ...
```

Comment:
remember to update this to just `dynatrace.com/synthetics-ids` as well.

Reply:
i think you maybe forgot to push the changes to these? marked as unresolved for now